CONTRIBUTING.md

```diff
@@ -1,20 +1,144 @@
-Thanks for wanting to contribute to Mozilla's Add-ons Linter! You rock! 😊
+Hi! Thanks for wanting to contribute to Mozilla's Add-ons Linter! You rock! 😊
 
-## Submitting a Pull Request
+The linter is used in various stages of [WebExtension](https://developer.mozilla.org/en-US/Add-ons/WebExtensions)
```

Comment:
This sentence is phrased a bit awkwardly and it makes it hard to parse. Also I'm pretty sure we're not supposed to say WebExtensions? I dunno. Saying "contributions are awesome" is a bit of a generic call-to-action as well, but I'd encourage people involved with extensions to relay their experience and get involved... I'd try:

```
This linter is used to help develop and publish [extensions](https://developer.mozilla.org/en-US/Add-ons/WebExtensions) for Firefox. You're an add-on developer, we would really value your contributions–no one knows add-on development and publishing better than an add-on developer!